main.go

```diff
@@ -196,13 +208,17 @@ func parseResult(file string, outputArray []outputData) (err error) {
 		inZoneTrafficScore := rowData.result.InZoneTraffic * 100
 		// use mean deviation to calcualte deviation score
 		deviationScore := 100.0 - rowData.result.MeanDeviation*100
+		// use number of EndpointSlices deviation to calculate sliceScore
+		numberOfOriginalSlices := math.Ceil(float64(rowData.endpoints) / endpointsPerSlice)
+		sliceScore := 100 - (float64(rowData.endpointSlices)-numberOfOriginalSlices)/numberOfOriginalSlices*100
 		// calculate total score based on two scores above
-		totalScore := 0.6*inZoneTrafficScore + 0.4*deviationScore
+		totalScore := 0.5*inZoneTrafficScore + 0.3*deviationScore + 0.2*sliceScore
```

Comment:
at some point we may want to define the weights of the scores as vars at the top of the file so it's easy to tweak them.

Reply:
added const at very beginning.